modules/programs/firefox.nix

```diff
@@ -106,8 +107,13 @@ let
       <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=UTF-8">
       <TITLE>Bookmarks</TITLE>
       <H1>Bookmarks Menu</H1>
+
       <DL><p>
-      ${bookmarkEntries}
+        ${bookmarkEntries}
+        <DT><H3 PERSONAL_TOOLBAR_FOLDER="true">Bookmarks Toolbar</H3>
+        <DL><p>
+          ${toolbarEntries}
+        </p></DL>
```

Comment:
You're missing a closing `</DT>` tag. Also, shouldn't this section be conditional on there being at least one toolbar entry?

Reply:
Hey thanks for the feedback. I was just going off of what was being done for the individual bookmarks, but looking into it, it seems that even Firefox doesn't include the closing DT tags.... so not sure what to do here. I've updated to make the toolbar section optional